app/controllers/api/v1/orders_controller.rb

```diff
@@ -7,6 +7,10 @@ def index
         collection(Order.all)
       end
 
+      def show
+        render :json => Order.find(params.require(:id))
```

Comment:
@syncrou This takes an id, while cancel_order and submit_order take in order_id. Should we be consistn=ent

Reply:
@syncrou so we need RBAC checks?

Reply:
@mkanoor In regards to the endpoint rails uses an `id` param in /<object>/id and a foreign key param in /<object>/object_id/<subcollection> so the design of the endpoint follows rails conventions.

Reply:
@mkanoor RBAC before_action was added.